modules/gitops-release-notes-1-6-2.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+
+[id="gitops-release-notes-1-6-2_{context}"]
+= Release notes for {gitops-title} 1.6.2
+
+{gitops-title} 1.6.2 is now available on {product-title} 4.8, 4.9, 4.10 and 4.11.
+
+[id="fixed-issues-1-6-2_{context}"]
+== Fixed issues
+
+The following issue have been resolved in the current release:
+
+* Before this update, subscription health check was marked *degraded* for missing *InstallPlan* when more than 5 operators were installed in a project. This update fixes the issue. link:https://issues.redhat.com/browse/GITOPS-2018[GITOPS-2018]
```

Comment:
```suggestion
* Before this update, the subscription health check was marked *degraded* for a missing *InstallPlan* when more than 5 Operators were installed in a project. This update fixes the issue. link:https://issues.redhat.com/browse/GITOPS-2018[GITOPS-2018]
```

Reply:
_IBM Style Guide_: Articles increase clarity and ease translation. The articles “a” and “an” are indefinite articles. Use them to refer to things in the
abstract or not in the particular. “An offering” or “a product” means any offering or any product, as opposed to a specific offering or product. “The” is a definite article. Use it to refer to a particular thing or things, meaning that “the offering” or “the product” is about that particular offering or product. Do not omit the articles a, an, and the.

Reply:
SSG: The term "Operator" in the context of Kubernetes is always capitalized to distinguish it from other types of operators, such as human or mathematical operators.